docs/release-notes/8.0/8.0.34-29.0.md

```diff
@@ -0,0 +1,31 @@
+# Percona XtraBackup 8.0.34-29 (2023-08-14)
+
+Get started with [Quickstart Guide for Percona XtraBackup](quickstart-overview.md).
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method](..//..//create-compressed-backup.md) is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becomes the default compression method for the [`--compress`](..//..//create-compressed-backup.md) option.
```

Comment:
Add a heading `### Compression`
Keep the note about ZSTD moving to default
Add a note that quicklz has been removed from --backup but still an option to --decompress older backups.

Reply:
updated